sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/ReceiptExtensions.java

```diff
@@ -108,10 +108,9 @@ public static USReceipt asUSReceipt(RecognizedReceipt receipt) {
     @SuppressWarnings("unchecked")
```

Comment:
if possible we can remove this unchecked?

Reply:
We cannot, using generics and this will throw error warnings when building the client.